crates/ruma-macros/Cargo.toml

```diff
@@ -1,14 +1,14 @@
 [package]
 categories = ["api-bindings", "web-programming"]
-description = "A procedural macro used by the ruma-events crate."
+description = "Procedural macro used by the ruma crates."
```

Comment:
```suggestion
description = "Procedurals macro used by the Ruma crates."
```

Reply:
Shouldn't the `s` be on `macro` instead of `procedural`?

Reply:
Oops, yeah 😄